kotlinx-coroutines-core/jvm/test/channels/ActorTest.kt

```diff
@@ -28,14 +27,14 @@ class ActorTest(private val capacity: Int) : TestBase() {
             expect(3)
         }
         actor as Job // type assertion
-        assertThat(actor.isActive, IsEqual(true))
-        assertThat(actor.isCompleted, IsEqual(false))
-        assertThat(actor.isClosedForSend, IsEqual(false))
+        assertEquals(true, actor.isActive)
```

Comment:
These are also easy to replace with `assertTrue` and `assertFalse`. Would it be better?